### ISSUE TITLE
Cannot add new shows, indexer search does not return results. 

### ISSUE THREAD

**Issue by mapleleafs89** [SickChill/SickChill#6245]
After the recent update, sickchill was failing to start due to the import issues described in the other thread. I read the steps to resolve the issue which was not related to code, but just leftover files that needed cleaning (thank you). This fixed the issue and sickchill starts and h as been fetching existing content successfully. My issue now is that no results are returned when I attempt to add a new show. There is not much info in the debug logs for this event. 

### Before submitting your issue:

Enable debug logging in SickChill settings, reproduce the error (be sure to disable after the bug is fixed)

Branch/Commit: master 518e33e628df975eb178c2a6b8541066818a2359
OS: CentOS 7
Version: v2020.04.20-7
Database Version: 44.1
Browser: Chrome
What you did: Add new show
What happened: No search results 
What you expected: Provided show options matching term
Logs:
```
2020-04-20 13:04:37 DEBUG WEBSERVER-ADDSHOWS :: Searching for Show with searchterm(s): ['Waco'] on Indexer: theTVDB
```

I also see a lot of these errors in the logs now after the update. 

```
2020-04-20 13:02:51 DEBUG SEARCHQUEUE-DAILY-SEARCH :: Check scene exceptions update failed. Unable to update from http://sickchill.github.io/scene_exceptions/scene_exceptions.json
2020-04-20 13:02:51 DEBUG SEARCHQUEUE-DAILY-SEARCH :: Requested a json response but response was not json, check the url: http://sickchill.github.io/scene_exceptions/scene_exceptions.json
```

**Comment by miigotu** [comment]
Looks like it doesnt have internet access, restart the machine maybe.

**Comment by damagu** [comment]
This was affecting me too after I followed the instructions in the other thread and a restart seemed to fix it.

**Comment by mapleleafs89** [comment]
Thanks @miigotu  and @damagu . I patched my server and rebooted, unfortunately its still only sickchill unable to reach out. Couchpotato, plex, jackett, everything have no issues with external connections.

Since SickChill isn't reporting the exact error code in debug logs, anything I can do to get some additional information from it? I don't know what method SickChill uses for it's get requests, but I can git the scene exceptions json URL using curl on the same system, though it's a 301 redirect to the https version. I am assuming SickChill or the library it's using for these requests would handle the 301 appropriately. As for the indexer returning no results, I don't know the URL it's trying to hit so can't test from host system.

Maybe I'm going down the wrong path considering the host clearly has internet access and there is something else the upgrade has left around that could impact specifically SickChill.

`curl -vvv http://sickchill.github.io/scene_exceptions/scene_exceptions.json
* About to connect() to sickchill.github.io port 80 (#0)
*   Trying 185.199.111.153...
* Connected to sickchill.github.io (185.199.111.153) port 80 (#0)
> GET /scene_exceptions/scene_exceptions.json HTTP/1.1
> User-Agent: curl/7.29.0
> Host: sickchill.github.io
> Accept: */*
>
< HTTP/1.1 301 Moved Permanently
< Server: GitHub.com
< Content-Type: text/html
< Location: https://sickchill.github.io/scene_exceptions/scene_exceptions.json
< X-GitHub-Request-Id: 241A:11A6:DCAC0:1288DC:5E9E3D84
< Content-Length: 162
< Accept-Ranges: bytes
< Date: Tue, 21 Apr 2020 00:26:39 GMT
< Via: 1.1 varnish
< Age: 58
< Connection: keep-alive
< X-Served-By: cache-mdw17321-MDW
< X-Cache: HIT
< X-Cache-Hits: 1
< X-Timer: S1587428799.095893,VS0,VE1
< Vary: Accept-Encoding
< X-Fastly-Request-ID: 68123f7ec589383f7dcbcc7edb1d2c8f0e1eb619
<
<html>
<head><title>301 Moved Permanently</title></head>
<body>
<center><h1>301 Moved Permanently</h1></center>
<hr><center>nginx</center>
</body>
</html>
* Connection #0 to host sickchill.github.io left intact
`

**Comment by miigotu** [comment]
Do you have pip installed?

**Comment by mapleleafs89** [comment]
> Do you have pip installed?

Yeah I do have pip installed.

```
[root@centos7 ~]# rpm -qa | grep -i pip
python2-pip-8.1.2-12.el7.noarch
python3-pip-9.0.3-7.el7_7.noarch
python34-pip-8.1.2-12.el7.noarch
```

**Comment by miigotu** [comment]
Quite possibly the API isn't working in your area right now also

**Comment by mapleleafs89** [comment]
> Quite possibly the API isn't working in your area right now also

Not sure if this is the same thetvdb API URL but I can hit it and get a 401 at least.

```
[root@centos7 ~]# curl https://api.thetvdb.com/
{"Error":"Not authorized"}
```

**Comment by mapleleafs89** [comment]
Some more logs. Can see it's at least reporting 200 on these api calls, but again for some reason it's not returning JSON or not handling it? Really strange.. I wish I could see the full request/response.

```
2020-04-20 21:02:11 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Fallback season pack search initiate
2020-04-20 21:02:11 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: URL did not return data
2020-04-20 21:02:11 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Requested a json response but response was not json, check the url: https://apibay.org/q.php
2020-04-20 21:02:11 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: GET URL: https://apibay.org/q.php?q=Last+Week+Tonight+with+John+Oliver+S07E09&cat=208%2C205 [Status: 200]
2020-04-20 21:02:10 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Search string: Last Week Tonight with John Oliver S07E09
2020-04-20 21:02:10 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: URL did not return data
2020-04-20 21:02:10 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Requested a json response but response was not json, check the url: https://apibay.org/q.php
2020-04-20 21:02:10 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: GET URL: https://apibay.org/q.php?q=tt3530232&cat=208%2C205 [Status: 200]
2020-04-20 21:02:09 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Search string: tt3530232
2020-04-20 21:02:09 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Search Mode: Episode
2020-04-20 21:02:08 DEBUG SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: GET URL: https://torrindex.net/static/main.js [Status: 200]
2020-04-20 21:02:08 INFO SEARCHQUEUE-MANUAL-278518 :: [ThePirateBay] :: Performing episode search for Last Week Tonight with John Oliver
```

**Comment by miigotu** [comment]
apibay is not thetvdb... That is thepiratebay...

**Comment by mapleleafs89** [comment]
> apibay is not thetvdb... That is thepiratebay...

Yeah so it seems the problem is not isolated to thetvdb, as we also saw these invalid json errors on the Github request.. All these external api calls are returning nothing/invalid json according to sickchill logs. Is there a specific Python package I could reinstall? Or I guess at this point I can try backing up sickchill DB and reinstall sickchill and restore the DB?

**Comment by miigotu** [comment]
Personally, I'm not using a clean environment. I removed my lib dir, and did `pip install -Ut lib -r requirements.txt` from my SC dir, so that I would get all the compiled c-extensions and speed up my SC (this is a secret haha)

I cant really recommend you doing this unless you know what you are doing, but SC __WILL* work then.

Alternatively, backing up and cleaning everything from the old install dir and then reinstalling might be of help.

**Comment by mapleleafs89** [comment]
Thanks, I ended up trying a fresh clone and copied my data over. To my surprise it still did not work.. so obviously it was something specific to my machine. I noticed in the systemctl status a deprecation warning for the openssl fips version, except my version was extremely old. The python version I was running was not manage by my package manager yum, so I guess it had not been updated in a long time (OpenSSL 1.0.1e-fips 11 Feb 2013). This also led me to #5347 due to the openssl deprecation warnings.

I switched to python 2.7.5 from 2.7.11 since the 2.7.5 on my machine is managed by yum and had more recently been updated and includes OpenSSL 1.0.2k-fips 26 Jan 2017. Couchpotato and the other programs also used this version of python which explains why they had worked... I neglected the version used by SC. I could have installed the latest 2.7.11 as well but this was a flip of the switch and all was working after restart, I will upgrade to a newer version later on.

Thanks for your help @miigotu
